Saule/Saule.csproj

```diff
@@ -41,6 +41,14 @@
     <CodeAnalysisIgnoreGeneratedCode>true</CodeAnalysisIgnoreGeneratedCode>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">

```

Comment:
We definitely don't want to pull in Entity Framework as a reference. Why do you think we need this?